website/documentation/050-tutorials/content/howto/shell-to-node/_index.md

```diff
@@ -0,0 +1,387 @@
+---
+title: Get a Shell to a Gardener Shoot Worker Node
+description: "Describes the methods for getting shell access to worker nodes."
+type: tutorial-page
+level: advanced
+index: 10
+category: Debugging
+scope: operator
+---
+
+# Get a Shell to a Kubernetes Node 
+
+To troubleshoot certain problems in a Kubernetes cluster, operators need access to the host of the Kubernetes node to troubleshoot 
+problems. This can be required if a node misbehaves or fails to join the cluster in the first place.
+With ssh access to the host, it is for instance possible to check the `kubelet` logs and interact with common tools such as `systemctl`and `journalctl`.
+
+The first section of this guide explores options to get a shell to the node of a Gardener Kubernetes cluster. 
+The options described in the second section do not rely on Kubernetes capabilities to get shell access to a node and thus can also be used if an instance failed to join the cluster.
+This guide only covers how to get access to the host, but does not cover troubleshooting methods.
+
+- [Shell access to an operational cluster node](#ssh-access-to-an-operational-cluster-node)
+  - [Gardener Dashboard](#gardener-dashboard) 
+  - [gardenctl shell](#gardenctl-shell) 
+  - [Gardener Ops Toolbelt](#gardener-ops-toolbelt)
+  - [Custom root pod](#custom-root-pod)
+- [SSH access to a node that failed to join the cluster](#ssh-access-to-a-node-that-failed-to-join-the-cluster)
+ - [gardenctl-ssh](#gardenctl-ssh) 
+ - [SSH using a manually created Bastion on AWS](#ssh-using-a-manually-created-bastion-aws) 
```

Comment:
The last TOC link doesn't work. Inspecting the id for the heading HTML element shows that it's bit strangely constructed. No idea how come.. But the working id is the one below:
```suggestion
 - [SSH using a manually created Bastion on AWS](#user-content-ssh-using-a-manually-created-bastion-on-aws) 
```